content/workshop/rapid-prototyping-with-slds/index.md

```diff
@@ -104,19 +130,48 @@ Feel free to add more components to the page and go crazy! You'll see the result
 
 ### 2. Directly from Chrome
 
-TODO: instructions to show how to edit views and CSS in the browser.
+1. Open <http://localhost:3000>
+1. Inspect any element on the page (**right click, inspect element**), this will open the Chrome DevTools
+1. In DevTools, go to the **Sources** tab
+1. **Right-Click** anywhere in the left pane and select **Add folder to workspace**
+1. **Find the design-system-starter-kit directory** (where it was cloned)
+1. Hit **Select**
+1. Click **Allow** to give the DevTools access to your drive
 
 {{% fluid_img "images/add-folder-to-workspace.gif" %}}
+
+Congrats, you now have access to your directory structure from within Chrome!
+
+Let's see what you can do:
+
+#### Editing pages in the browser
+
 {{% fluid_img "images/edit-in-browser.gif" %}}
+
+#### Editing styles in the browser
+
 {{% fluid_img "images/edit-scss-in-browser.gif" %}}
 
 ## From Sketch to the browser (section title TBD)
 
-TODO: instructions to copy - paste some components into a view.
+Let's do some copy and pasting! Errr… some coding!
+
+In the `src/views` directory, there's a file called **`workshop.html`**. That's where we're going to make the magic happen.
+
+TODO: More instructions to copy - paste some components into a view.
```

Comment:
Maybe we don't need more instructions if we have comments in the `workshop.html` file? Just a thought. :-)